nx/lib/nx.ex

```diff
@@ -9167,6 +9167,93 @@ defmodule Nx do
     end
   end
 
+  @doc """
+  Finds the variance of a tensor.
+
+  The variance is the average of the squared deviations from the mean.
+  The mean is typically calculated as sum(x) / N, where N = total of elements.
+  If, however, ddof is specified, the divisor N - ddof is used instead.
+
+  ## Examples
+
+  iex> Nx.variance(Nx.tensor([[1, 2], [3, 4]]))
```

Comment:
Please indent all code examples by four spaces and remove the new line between the expression and the result. :)